prediction_market_agent/agents/think_thoroughly_agent/think_thoroughly_agent.py

```diff
@@ -201,6 +256,7 @@ def generate_final_decision(
                 ),
                 "number_of_scenarios": len(scenarios_with_probabilities),
                 "scenario_to_assess": question,
+                "correlated_markets": [r.dict() for r in correlated_markets],
```

Comment:
I think dicts were messing up the prompt a little, wdyt about doing it in a human-readable way as above?

![Screenshot by Dropbox Capture](https://github.com/gnosis/prediction-market-agent/assets/15619339/479148fa-23ef-4db7-b6ba-e2e28ef1a626)